test/integration/install.py

```diff
@@ -27,20 +31,44 @@ def __init__(self, path):
 			print("WARNING: bitness misalignment between interpreter (%s) and driver (%s): testing will likely fail" %
 					(bitness_py, bitness_driver))
 
+		self._driver_name = "%s (%sbit)" % (DRIVER_BASE_NAME, bitness_driver)
+
 	def _install_driver_win(self, ephemeral):
+		# remove any old driver (of tested bitness)
+		name_filter = "name = '%s'" % self._driver_name
+		with psutil.Popen(["wmic", "/INTERACTIVE:OFF", "product", "where", name_filter, "call", "uninstall"],
+				stdout=PIPE, stderr=PIPE, universal_newlines=True) as p:
+			try:
+				p.wait(WMIC_UNINSTALL_TIMOUT)
+			except psutil.TimeoutExpired as e:
+				try: p.kill()
+				except: pass
+				raise Exception("wmic uninstallation killed after %s seconds" % WMIC_UNINSTALL_TIMOUT)
+
+			assert(p.returncode is not None)
+			out, err = p.communicate()
+			if (out): print(out)
+			if (err): print(err)
+			if p.returncode:
+				print("ERROR: driver wmic-uninstallation failed.")
```

Comment:
If it ever did fail would it be useful to know the exact value of the return code?  (Maybe not as I guess the log tells you more.)

Reply:
The logs should hopefully be relevant, but having the code can only help, so I've added it. Thanks.